magento/magento2
# Fixes issue #7626 - frontend_labels always empty

### Description
No matter how an attribute is created, frontend_labels is always null...

Magento 2.1.2
PHP 7.0.8


### Fixed Issues (if relevant)

1. magento/magento2#7626: frontend_labels always empty

### Manual testing scenarios
Create product atrribute
Call the attribute
Call attribute products/attributes/auto_artikelnummer

Actual result

{ "is_wysiwyg_enabled": false, "is_html_allowed_on_front": false, "used_for_sort_by": false, "is_filterable": false, "is_filterable_in_search": false, "is_used_in_grid": false, "is_visible_in_grid": false, "is_filterable_in_grid": false, "position": 0, "apply_to": [], "is_searchable": "1", "is_visible_in_advanced_search": "1", "is_comparable": "0", "is_used_for_promo_rules": "0", "is_visible_on_front": "1", "used_in_product_listing": "0", "is_visible": true, "scope": "global", "attribute_id": 163, "attribute_code": "auto_artikelnummer", "frontend_input": "text", "entity_type_id": "4", "is_required": false, "options": [], "is_user_defined": true, "default_frontend_label": "Artikelnummer", "frontend_labels": null, "backend_type": "varchar", "is_unique": "0", "validation_rules": [] }

frontend_labels is always null
validation_rules is always an empty array

Expected result

{ "is_wysiwyg_enabled": false, "is_html_allowed_on_front": false, "used_for_sort_by": false, "is_filterable": false, "is_filterable_in_search": false, "is_used_in_grid": false, "is_visible_in_grid": false, "is_filterable_in_grid": false, "position": 0, "apply_to": [], "is_searchable": "1", "is_visible_in_advanced_search": "1", "is_comparable": "0", "is_used_for_promo_rules": "0", "is_visible_on_front": "1", "used_in_product_listing": "0", "is_visible": true, "scope": "global", "attribute_id": 163, "attribute_code": "auto_artikelnummer", "frontend_input": "text", "entity_type_id": "4", "is_required": false, "options": [], "is_user_defined": true, "default_frontend_label": "Artikelnummer", "frontendLabels": [{ "storeId": 0, "label": "Artikelnummer" }], "backend_type": "varchar", "is_unique": "0", "validation_rules": [] }

This fix just applies to the *frontend_labels* since the validations rules are only used for customer attributes and there is no repository for customer attributes exposed as service.

### Contribution checklist
